src/constants.ts

```diff
@@ -155,6 +155,11 @@ export const AUTHORITY_TOKEN_DATA = TOKEN_AUTHORITY_MASK | 1;
  */
 export const HATHOR_TOKEN_CONFIG = { name: 'Hathor', symbol: 'HTR', uid: '00' };
```

Comment:
question: Would it be a good idea to keep the `HATHOR_TOKEN_CONFIG` unchanged and add all information about the network's native token to the `config.ts`?

All references to `HATHOR_TOKEN_CONFIG`, not only its `uid` would need to be changed by that.

Reply:
As discussed, only the native token name and symbol may change on private networks and side-dags.
Leanving the `NATIVE_TOKEN_UID` a constant